FilmFestivalLoader/Shared/planner_interface.py

```diff
@@ -537,17 +552,22 @@ def write_films(self):
         public_films = [f for f in self.films if self.film_can_go_to_planner(f.filmid)]
         if len(self.films):
             with open(self.films_file, 'w') as f:
-                f.write(self.films[0].film_repr_csv_head())
+                f.write(Film.film_repr_csv_head())
                 for film in public_films:
                     f.write(repr(film))
             print(f'Done writing {len(public_films)} of {len(self.films)} records to {self.films_file}.')
         else:
             print('No films to be written.')
-        if len(self.films):
+
+    def write_film_ids(self):
+        film_id_count = len(self.film_id_by_url)
+        if film_id_count > 0:
             with open(self.filmids_file, 'w') as f:
-                for film in self.films:
-                    f.write(film.filmid_repr())
-            print(f'Done writing {len(self.films)} records to {self.filmids_file}.')
+                for url, film_id in self.film_id_by_url.items():
+                    title = self.title_by_film_id[film_id]
+                    text = ";".join([str(film_id), title.replace(";", ".,"), url])
```

Comment:
Please consider to start using `import csv`.

Reply:
Won't do now, needs a CSV module at the reading side (Planner) as well, a bit out of scope.

Created #290 though, so it will be done somewhere in the future.